src/main/resources/reference.conf

```diff
@@ -58,7 +61,11 @@ resourceTypes = {
       can-compute = {
         roleActions = ["compute"]
       }
+      can-catalog = {
+        roleActions = ["catalog"]
+      }
     }
+    authDomainConstrainable = true
```

Comment:
does this need to be added to test's reference.conf as well? 🤔

Reply:
no i don't think so. why do you ask?

Reply:
nothing specific, just wonder if it should be added in test's reference.conf and possibly be tested

Reply:
it's already in there for some test-specific resource type